src/contractor/contractor.cpp

```diff
@@ -387,19 +388,17 @@ parse_turn_penalty_lookup_from_csv_files(const std::vector<std::string> &turn_pe
         const auto file_id = idx + 1; // starts at one, zero means we assigned the weight
         const auto filename = turn_penalty_filenames[idx];
 
-        std::ifstream turn_penalty_file{filename, std::ios::binary};
-        if (!turn_penalty_file)
-            throw util::exception{"Unable to open turn penalty file " + filename};
-
+        storage::io::FileReader turn_penalty_file_reader(std::string(filename),
```

Comment:
Well this is weird. Earlier, both this and the segment_speed_filenames[idx] were failing to compile because filename was a char* and not a string. And therefore would not compile without being cast as a string. Now it seems to be compiling just fine. So I'll remove it.